user/views.py

```diff
@@ -1,3 +1,37 @@
-from django.shortcuts import render
+import json
+import jwt
+import bcrypt
 
-# Create your views here.
+from my_settings import SECRET_KEY
+
+from django.views import View
+from django.http  import JsonResponse
+
+from user.models import User
+
+class LoginView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            email    = data['email']
+            password = data['password']
+
+            exist_user = User.objects.filter(email=email).exists()
+            if not exist_user:
+                return JsonResponse({'message':'INVALID_USER'}, status = 400)
+            
+            user_info      = User.objects.filter(email=email)
```

Comment:
```suggestion
            user_info      = User.objects.get(email=email)
```
유니크 가정으로 요건 그냥 get하는게 아래에서 first()안써도 될 것 같아요!